src/openloco/input/keyboard.cpp

```diff
@@ -417,23 +420,97 @@ namespace openloco::input
         input::set_flag(input_flags::viewport_scrolling);
     }
 
+    static void PngWriteData(png_structp png_ptr, png_bytep data, png_size_t length)
+    {
+        auto ostream = static_cast<std::ostream*>(png_get_io_ptr(png_ptr));
+        ostream->write((const char*)data, length);
+    }
+
+    static void PngFlush(png_structp png_ptr)
+    {
+        auto ostream = static_cast<std::ostream*>(png_get_io_ptr(png_ptr));
+        ostream->flush();
+    }
+
     // 0x00452667
     static int16_t saveScreenshot()
     {
-        registers regs;
+        void* buffer = malloc(0x10000);
+        if (buffer == nullptr)
```

Comment:
This buffer is unused, right?

Reply:
Probably just a remnant of the BMP decompilation I thought of decompiling before.